builds/android/app/src/main/java/org/easyrpg/player/game_browser/GameScanner.java

```diff
@@ -94,8 +94,20 @@ private void scanGames(){
             first_directory = false;
         }
 
-        // Sort the games list
-        Collections.sort(gameList);
+        // Sort the games list : alphabetically ordered, favorite in first
```

Comment:
This is ugly. Should be possible to rewrite `compareTo` to do this.

Reply:
This is indeed done in GameInformation.

Reply:
I mean you do not need 2 lists and sort them each on their own.

Reply:
btw the compare logic could be made more local (where sort is called) by using an inner class or a lambda expresion: https://www.baeldung.com/java-8-sort-lambda

Reply:
Because compareTo have been rewrite in GameInformation, could you tell me why it's "ugly" and how to improve it?

Reply:
What he means is that you can just use one list and do a sort with multiple conditions. Is also much more efficient.

Reply:
Since GameInformation knows if a game is favourited, you can add that in the comparison.

I am a bit rusty in java, but you get the idea:

```java
	public int compareTo(GameInformation gameInformation) {
		if (this.isFavorite() && ! gameInformation.isFavorite())
			return 1;
		if (! this.isFavorite() && gameInformation.isFavorite())
			return -1;
		return this.title.compareTo(gameInformation.title);
	}
```

Reply:
I have done 2 separate sort because if I do one only sort, it won't put the favorite game in first.

Reply:
Also, please note that 2 lists could be usefull for the future. Example if we want to have a "favorite" separate from other game.

Reply:
Oh, our messages have been intertwined, please ignore the order.